parser.y

```diff
@@ -11932,4 +11934,33 @@ EncryptionOpt:
 		}
 		$$ = $1
 	}
+
+/*******************************************************************************
+ * TABLE table_name [ORDER BY column_name] [LIMIT number [OFFSET number]]
+ ******************************************************************************/
+TableStmt:
+	"TABLE" TableName OrderByOptional TableStmtLimit
+	{
+		st := &ast.TableStmt{Table: $2.(*ast.TableName)}
+		if $3 != nil {
+			st.OrderBy = $3.(*ast.OrderByClause)
+		}
+		if $4 != nil {
+			st.Limit = $4.(*ast.Limit)
+		}
+		$$ = st
+	}
+
+TableStmtLimit:
+	{
+		$$ = nil
+	}
+|	"LIMIT" LimitOption
+	{
+		$$ = &ast.Limit{Count: $2.(ast.ExprNode)}
+	}
+|	"LIMIT" LimitOption "OFFSET" LimitOption
+	{
+		$$ = &ast.Limit{Offset: $4.(ast.ExprNode), Count: $2.(ast.ExprNode)}
+	}
```

Comment:
please just use `SelectStmtLimit`. while the documentation in https://dev.mysql.com/doc/refman/8.0/en/table.html only mentions `LIMIT limit [OFFSET offset]`, in fact the legacy form `LIMIT offset, limit` does work in TABLE statements.

Reply:
Yeah, I found that too. I've not checked `"FETCH" FirstOrNext FetchFirstOpt RowOrRows "ONLY"` branch in the `SelectStmtLimit`, if this branch is not allowed, should I create the new `TableStmtLimit` or just use the `SelectStmtLimit`?

Reply:
still just use `SelectStmtLimit`. 

`TABLE t1 FETCH FIRST ROW ONLY` is a standard ISO syntax and also supported in PostgreSQL, so there's no reason to reject it.

Reply:
Ok, I'll fix it